web_src/less/_base.less

```diff
@@ -1,85 +1,57 @@
-/* lato-regular - latin */
+/* roboto-regular - latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext */
 @font-face {
-    font-family: 'Lato';
+    font-family: 'Roboto';
     font-style: normal;
     font-weight: 400;
-    src: url('../vendor/assets/lato-fonts/lato-v16-latin-regular.eot'); /* IE9 Compat Modes */
     src:
-        local('Lato Regular'),
-        local('Lato-Regular'),
-        /* IE6-IE8 */
-        url('../vendor/assets/lato-fonts/lato-v16-latin-regular.eot?#iefix') format('embedded-opentype'),
-        /* Super Modern Browsers */
-        url('../vendor/assets/lato-fonts/lato-v16-latin-regular.woff2') format('woff2'),
-        /* Modern Browsers */
-        url('../vendor/assets/lato-fonts/lato-v16-latin-regular.woff') format('woff'),
-        /* Safari, Android, iOS */
-        url('../vendor/assets/lato-fonts/lato-v16-latin-regular.ttf') format('truetype'),
-        /* Legacy iOS */
-        url('../vendor/assets/lato-fonts/lato-v16-latin-regular.svg#Lato') format('svg');
-}
-
-/* lato-italic - latin */
+        local('Roboto'),
+        local('Roboto-Regular'),
+        /* Chrome 26+, Opera 23+, Firefox 39+ */
+        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff2') format('woff2'),
+        /* Chrome 6+, Firefox 3.6+, IE 9+, Safari 5.1+ */
+        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff') format('woff');
```

Comment:
Would put comments like this, which will help us find those IE compats once its time to remove. No need to comment the woff2.

``` css
        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff2') format('woff2'),
        /* compat: IE11 */
        url('../vendor/assets/roboto-fonts/roboto-v20-latin-ext_cyrillic-ext_latin_greek_vietnamese_cyrillic_greek-ext-regular.woff') format('woff');
```

Reply:
Comments are generated by that font download code, it's same also as for lato

Reply:
Which code? Do we have a font updating script somewhere?

Reply:
This code: https://github.com/go-gitea/gitea/pull/9803/files#diff-3635838a3c33d79a155c7526989c5577R126

Reply:
It's manually copied over